From 207aa307a1234343f2279db58080a139145a83b5 Mon Sep 17 00:00:00 2001
From: Guan Xuetao <gxt@mprc.pku.edu.cn>
Date: Mon, 28 Nov 2011 16:50:09 +0800
Subject: [PATCH 090/641] unicore64: remove redundant set_fs(USER_DS)

The address limit is already set in flush_old_exec() so this
set_fs(USER_DS) is redundant.

Signed-off-by: Guan Xuetao <gxt@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/processor.h |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/unicore64/include/asm/processor.h b/arch/unicore64/include/asm/processor.h
index f2a190d..ff8a330 100644
--- a/arch/unicore64/include/asm/processor.h
+++ b/arch/unicore64/include/asm/processor.h
@@ -46,7 +46,6 @@ extern unsigned long get_wchan(struct task_struct *p);
 #define start_thread(regs, pc, sp)					\
 ({									\
 	unsigned long *stack = (unsigned long *)sp;			\
-	set_fs(USER_DS);						\
 	memset(regs->uc64_regs, 0, sizeof(regs->uc64_regs));		\
 	regs->UC64_ASR = ASR_MODE_USER;					\
 	regs->UC64_R31 = pc & ~1;	/* pc */                        \
-- 
1.7.9.5

